### PR TITLE
Simplify Recipe Code

### PR DESCRIPTION
**Pull request by Alek99**

### CODE REVIEW

**Comment by tgberkeley** [comment]
When we click the recipes tab it should automatically open the overview page (http://reflex.dev/docs/recipes-overview/) so we can see all the recipes

**Comment by tgberkeley** [comment]
The accordions should be in alphabetical order ideally

File: pcweb/components/docpage/sidebar/sidebar.py [hunk 1]
```diff
@@ -266,42 +266,49 @@ def get_prev_next(url):
...
+            rx.button(
+                rx.icon(
+                    tag=icon,
+                    color="white",
+                    fill="rgba(255, 255, 255, 0.25)",
+                    size=20,
+                ),
+                height="35px",
+                width="35px",
+                padding="0px",
```
Comment (picklelo) [inline pcweb/components/docpage/sidebar/sidebar.py:281]: Is this needed? Does button set default padding?

File: pcweb/components/docpage/sidebar/sidebar.py [hunk 2]
```diff
@@ -266,42 +266,49 @@ def get_prev_next(url):
...
-                fill="rgba(255, 255, 255, 0.25)",
-                size=20,
+def sidebar_category(name, url, icon, color, index):
+    return rx.link(
+        rx.flex(
+            rx.button(
+                rx.icon(
+                    tag=icon,
+                    color="white",
+                    fill="rgba(255, 255, 255, 0.25)",
```
Comment (picklelo) [inline pcweb/components/docpage/sidebar/sidebar.py:276]: What's the difference between color and fill?

File: pcweb/components/docpage/sidebar/sidebar.py [hunk 3]
```diff
@@ -373,10 +380,10 @@ def sidebar_comp(
     width: str = "100%",
 ):
     return rx.flex(
-        sidebar_category("Learn", "graduation-cap", "jade", 0),
-        sidebar_category("Components", "layout-panel-left", "blue", 1),
-        sidebar_category("Recipes", "scan-text", "indigo", 2),
-        sidebar_category("API Reference", "book-text", "violet", 3),
+        sidebar_category("Learn", "/docs/getting-started/introduction/", "graduation-cap", "jade", 0),
```
Comment (picklelo) [inline pcweb/components/docpage/sidebar/sidebar.py:383]: Don't use hard-coded links

**Review by picklelo — CHANGES_REQUESTED** [review]

File: pcweb/components/docpage/sidebar/sidebar.py (hunk 2 above)
Comment (Alek99) [reply pcweb/components/docpage/sidebar/sidebar.py:276]: For the icons color is the outline of the icon fill is the inside color, you can see slight differences if you check out the deployed sidebar item

**Review by picklelo — APPROVED** [review]

File: pcweb/components/docpage/sidebar/sidebar.py (hunk 1 above)
Comment (Alek99) [reply pcweb/components/docpage/sidebar/sidebar.py:281]: This was same as before just refactored the function yes its needed we use more than the default spacing

**Pull request merged**